### PR TITLE
**Gallery with Images**

### PR DESCRIPTION
**Pull request by knhrvs**
***crop lang yung images okay lang ba yon??

### CODE REVIEW

File: src/pages/gallery.astro [hunk 1]
```diff
@@ -7,53 +7,53 @@ import Carousel from '../components/CarouselGallery.tsx'
...
 
 
-const images2k18 = {
+const images2k17 = {
   desktopView: [
       {
-      backgroundImageFirst: 'https://via.placeholder.com/300x300/0000FFC',
-      backgroundImageSecond: 'https://via.placeholder.com/300x300/FFFFFFF',
-      backgroundImageThird: 'https://via.placeholder.com/300x300/0000FFC',
+      backgroundImageFirst: 'images/gallery/csconk217/During4.jpg',
```
Comment (Simperfy) [inline src/pages/gallery.astro:13]: Images not found on deployment
image

Solution: add prefix to all assets path with `/`

```diff
-backgroundImageFirst: 'images/gallery/csconk217/During4.jpg',
+backgroundImageFirst: '/images/gallery/csconk217/During4.jpg',
```

**Review by Simperfy — CHANGES_REQUESTED** [review]

File: src/pages/gallery.astro (hunk 1 above)
Comment (knhrvs) [reply src/pages/gallery.astro]: okay na

**Comment by knhrvs** [comment]
**may na encounter pala akong problem sa dark mode

**Comment by Simperfy** [comment]
> **may na encounter pala akong problem sa dark mode

Huwag mo na pansinin iyan, ireremove na natin dark mode.

**Review by DauntlessDev — CHANGES_REQUESTED** [review]
I asked for feedback from the co-head and head and they said that the picture's proportion gets distorted in some sizes of screen, it is better to make it not scale and just center crop the picture if the picture size doesn't match the screen size. Thanks ! 🤩

**Review by Simperfy — APPROVED** [review]

**Comment by knhrvs** [comment]
> I asked for feedback from the co-head and head and they said that the picture's proportion gets distorted in some sizes of screen, it is better to make it not scale and just center crop the picture if the picture size doesn't match the screen size. Thanks ! 🤩

i just updated na yung mobile view ng galley and okay na sya di na distorted yung images, may nakalimutan lang pala ako alisin. Then another concern ko pala is diba may images na di sakto kaya naglagay ako ng border sa taas and baba nya okay lang ba yon?

**Review by DauntlessDev — APPROVED** [review]

**Pull request merged**